gui/requirements.txt

```
@@ -1,5 +1,5 @@
 # Core
-streamlit~=1.29.0
+streamlit
```

```suggestion
streamlit==1.32.0
```
please pin the dependency with specific version

---

Fix in [3a5e361](https://github.com/openvinotoolkit/datumaro/pull/1334/commits/3a5e361550294097c4afe167f8028d919182c76d)